demos/embedded/src/main/java/org/eclipse/jetty/demos/Http2Server.java

```diff
@@ -98,12 +99,11 @@ public static void main(String... args) throws Exception
         server.addConnector(http);
 
         // SSL Context Factory for HTTPS and HTTP/2
-        Path keystorePath = Paths.get("src/main/resources/etc/keystore.p12").toAbsolutePath();
-        if (!Files.exists(keystorePath))
-            throw new FileNotFoundException(keystorePath.toString());
         SslContextFactory.Server sslContextFactory = new SslContextFactory.Server();
-        sslContextFactory.setKeyStorePath(keystorePath.toString());
-        sslContextFactory.setKeyStorePassword("storepwd");
+        sslContextFactory.setKeyStore(TestKeyStoreFactory.getServerKeyStore());
```

Comment:
the only problem I have here and for all similar changes: we do not test anymore the same configuration (e.g `setKeyStorePath`)